Dockerfiles/cluster-agent/README.md

```diff
@@ -27,9 +27,26 @@ Therefore, you will need to have docker installed on your system.
 Start by creating the binary by running `inv -e cluster-agent.build`. This will add a binary in `./bin/datadog-cluster-agent/`
 Then from the current folder, run `inv -e cluster-agent.image-build`.
 
-
 ## Running the DCA with Kubernetes
 
+### Security premise
+<a name="security-premise"></a>
+
+You can create the secret that will be used for your agents to communicate with the DCA. 
+You can modify the value in [this file](/manifests/cluster-agent/dca-secret.yaml) then create it:
```

Comment:
Same here (cf RFC 2119). But sometimes it's easier just to get rid of the `you can` and use the imperative i.e:

instead of:
`You can modify the value in [this file](/manifests/cluster-agent/dca-secret.yaml) then create it:`
use: 
`Modify the value in [this file](/manifests/cluster-agent/dca-secret.yaml) then create it:`

------ 

When adding a link, you should avoid [here] / [this page] / [this file] placeholders, instead give directly the title of the page or of the file.

So the final sentence would be:

`Modify the value in [the dca-secret.yaml file](/manifests/cluster-agent/dca-secret.yaml) then create it:`